src/sentry/static/sentry/app/views/organizationDiscover/styles.jsx

```diff
@@ -7,12 +7,12 @@ import space from 'app/styles/space';
 
 import {Panel, PanelItem} from 'app/components/panels';
 import NavTabs from 'app/components/navTabs';
-import TextField from 'app/components/forms/textField';
 import Link from 'app/components/link';
 
 const FOOTER_HEIGHT = 87;
 const HEADER_HEIGHT = 60;
 const TABS_HEIGHT = 55;
+const QUERY_FIELDS_HEIGHT = 592;
```

Comment:
Can we do these layouts without fixed heights?

Reply:
I don't think there's any easy way to do this, since the popover is an absolutely positioned element and the height of the page depends on the results being returned. My plan is to remove the body scrolling from this page and treat the left and right panels as independently scrolling elements. I think we can remove this then since the element will have 100% height in that case.